PostController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
```

Comment:
Method name should be wrapped inside controller class.
Here the File Name :  PostController.php 
Module  : Aligent_ExamPart1

class Aligent_ExamPart1_PostController extends Mage_Core_Controller_Front_Action {